### ISSUE TITLE
I want Blokada to unblock system updaters by default

### ISSUE THREAD

**Issue by Dasmark** [blokadaorg/blokada#502]
I'm on Xiaomi ROM and it took me a while to figure out that I was unable to update my phone, because Blokada was by default blocking access to com.android.updater and com.xiaomi.discover. I'm pretty sure other people have this issue as well, so it might be worth considering unblocking the system update apps by default to avoid problems like this.

**Comment by PrintableCharacter** [comment]
Hey there,
There are lots and lots of ROMs and phone producers out there. Going through every single ROM and checking for important apps is simply impossible.
Still: Thanks for pointing it out. This might help :)

**Comment by tanmay-bhat** [comment]
Yes, as there are lot of variants it would be difficult to do. So here are some workarounds.
1. whitelist Updater app 
2. allow com.xiaomi.discover, com.android.updater in allowed hosts.
 have a good day.

**Comment by Dasmark** [comment]
I already put them into allowed hosts, that's why I know it was the problem.
Although it might be hard to catch all updater apps in the beginning, wouldn't it make sense to start? I'm pretty sure you can cover >90% of all updater apps easily which is already very good
You're already whitelisting apps like Signal, Waze, Duo, Drive and Plex, so why not start adding the updaters?

**Comment by rpeter85** [comment]
By default, they aren't allowed because it depends on users' preference. Some of them want to block Xiaomi hosts, some of them don't want to.
Additionally, checking every manufacturer's (Samsung, Sony, Xiaomi, Huawei, Amazon, etc-etc) apps and updaters takes a lot of effort to track and keep the lists up-to-date.
